app/containers/Wallet/SubHeader.jsx

```diff
@@ -34,6 +35,7 @@ const SubHeader = ({
     stakedInCurrentPeriod >= stakedInNextPeriod
       ? balance - stakedInCurrentPeriod
       : balance - stakedInNextPeriod;
+  const domainPolygonScan = process.env.BLOCKCHAIN_EXPLORERE_URL;
```

Comment:
`process.env.BLOCKCHAIN_EXPLORERE_URL` should be moved to some config as well

Reply:
Leaving the variable in the .env file would be wrong? The value of the variable must be different for the test and prod environment

Reply:
Created story to work on that refactoring separately - https://peerania.atlassian.net/browse/PEER20-352
It can be left as is in this PR.